packages/orchestrator-ui-components/src/components/WfoInlineNoteEdit/WfoInlineNoteEdit.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import type { ChangeEvent, FC } from 'react';
+
+import { EuiInlineEditText } from '@elastic/eui';
+
+import { useStartProcessMutation } from '@/rtk/endpoints/forms';
+import type { Subscription } from '@/types';
+
+interface WfoInlineNoteEditProps {
+    value: Subscription['note'];
+    subscriptionId?: Subscription['subscriptionId'];
+}
+
+export const WfoInlineNoteEdit: FC<WfoInlineNoteEditProps> = ({
+    value,
+    subscriptionId,
+}) => {
+    const [startProcess] = useStartProcessMutation();
+    const triggerNoteModifyWorkflow = () => {
+        const noteModifyPayload = [
+            { subscription_id: subscriptionId },
+            { note },
+        ];
+        startProcess({
+            workflowName: 'modify_note',
+            userInputs: noteModifyPayload,
+        });
+    };
```

Comment:
Move this block under the `useState`

Reply:
Please use the code suggestion option for this preference related suggestions.